guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
 ## Research papers
 
 ###### [Non-adoption of crypto-assets: Exploring the role of trust, self-efficacy, and risk](https://voskart.de/pdf/crypto_adoption.pdf)
```

Comment:
I would suggest ordering each resource in alphabetical order based on their title.

Reply:
Papers are ordered by date, newest at the top.